drivers/rilmodem/rilutil.c

```diff
@@ -50,6 +50,248 @@ struct ril_util_sim_state_query {
 	GDestroyNotify destroy;
 };
 
+static const struct {
+	int mcc;
+	const char *iso;
+} mcc_iso_code_map[] = {
+	{202, "gr"},	/* Greece */
```

Comment:
Two comments about this table:

1. Country ISO codes are usually uppercase ( eg. "NL" vs. "nl" ).

2. This is yet another MCC table we're adding to our code ( the first is in the mnclength plugin ).  Perhaps a better approach would be to create a common static MCC table that includes ISO codes and mnc lengths, or at least leverage the MCC-to-ISO table in the mnclength plugin?  That way, MCC changes ( which admittedly are rare ) are made in once place.

3. Finally, while working on automatic time zone detection, a similar table is created that also includes the full country name.  Perhaps we should do the same and move the full names out of the comments and into the table?